src/components/data-exploration/embedding/Embedding.jsx

```diff
@@ -281,25 +285,32 @@ const Embedding = (props) => {
       {renderExpressionView()}
       {
         data ? (
-
           <Scatterplot
             cellOpacity={0.8}
-            cellRadiusScale={0.1}
+            cellRadius={cellRadius}
+            setCellHighlight={updateCellsHover}
+            // setCellSelection={updateCellsHover}
+            theme='light'
             uuid={embeddingType}
-            view={view}
+            viewState={view}
+            updateViewInfo={updateCellCoordinates}
             cells={convertCellsData(data, cellSetHidden, cellSetProperties)}
             mapping='PCA'
-            selectedCellIds={selectedCellIds}
+            cellSelection={selectedCellIds}
             cellColors={
               (selectedCell)
                 ? new Map(Object.entries({ ...cellColors, [selectedCell]: [0, 0, 0] }))
                 : new Map(Object.entries(cellColors))
             }
-            updateStatus={updateStatus}
-            updateCellsSelection={updateCellsSelection}
-            updateCellsHover={updateCellsHover}
-            updateViewInfo={updateCellCoordinates}
-            clearPleaseWait={clearPleaseWait}
+            setViewState={({ zoom, target }) => {
+              setCellRadius(cellRadiusFromZoom(zoom));
+
+              setView({ zoom, target });
+            }}
+            getExpressionValue={() => { }}
+            getCellIsSelected={() => { }}
```

Comment:
These are functions that the scatterplot crashes if it doesn't receive them but are not really essential, so we had to fill them in with empty functions